### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,10 +7,12 @@ env:
     - REGISTRY_USER=praekeltorgdeploy
     - secure: "PZ+6VKOiC45K0fKgrRn8UAzxDvJ4fkkuM53VO5htez0hUt1w7zL0nApbbg+9J81vSHdHyAv6N/ln2Yn9mysHB0d6x9LUjYKFVCI+XnZTTv0QqiykrDR/CVSWiKVkbCjoRCPvQsOTx47Hb5WFpCUp3EmUpfJgJ9w9BNlYaEvJZYF1BZCByb3ATZ5+DWSM4AczaW7Z4YUivF7lLsMOgjtxwk7fTjI1Ufel9yD43aVAaSBai1jTGyHTm1k7Aez1xqT9HHkEkrZ1THuV08vzLaCtjqBirANLj7FPx0/W2pJOp4DGQio0c86lpjm+PCrvHg1syqZzY6M/mtgZ5KlIhVVk/7bMe/MzZAg8oJV2ndzKvkVYK2RYohY+CXuGWeoVDYLEMZc6lJXDrWufe01Alcu/BZ9ZwDD5K73sYbd/wLKk5LXizHGGN8JIwAILD8xwNxx6d23z8r2nlDjwys5ujhj3SPxwSfutJrvU6RLQGSh7ExYz2K6KJ68u4Mf26F+iXiqu+3YLZ8NQMcrOJtcKuCp0WfO88xGsvZB1zPb1x0Ee1e/3LcUAreHRQGXmYq6P1N9zR7gipHfrlfUiFsOH0O1vovOsSlz6gPhhREeqvcVldFlYZBKK8ibQ9O2MOM2HyBh2ICfrbFLV8DvcH5sQGwwNaauFog/GHunfKQScuP12EZU="
   matrix:
-    - VERSION=6 TAG_LATEST=1 PYTHON_TAG=py2.7 PYTHON_LATEST=1
+    - VERSION=7 TAG_LATEST=1 PYTHON_TAG=py2.7 PYTHON_LATEST=1
```
Comment (JayH5) [inline .travis.yml]: Do you need a Python 2 image for Molo 7?

Comment (codiebeulaine) [reply .travis.yml]: yeah we still support both

Comment (JayH5) [reply .travis.yml]: Do we want to push people to using Python 3 by setting `PYTHON_LATEST=1` for the Python 3 image?

Comment (codiebeulaine) [reply .travis.yml]: we could, I'll put it as a todo on our backlog. We have some maintenance things coming up. I'm not sure if GEM supports python 3 even though molo and plugins do

**Pull request merged**